### ISSUE TITLE
finetuning result is so bad when using my pretrained checkpoints on QVhighlights.

### ISSUE THREAD

**Issue by fake-warrior8** [Yui010206/SeViLA#9]
Hi, I used your [pre-trained SeViLA localizer checkpoints](https://huggingface.co/Shoubin/SeViLA/resolve/main/sevila_pretrained.pth) on QVHighlights to fine-tune NExT-QA and got similar NExT-QA results as in your paper (73.2 vs. 73.8). However, when I used your script to first pretrain a sevila localizer and then finetune NExT-QA using my pretrained SeViLA localizer checkpoints, I got only an accuracy of 45 in the first epoch (71 using the checkpoints your gave). I found that your checkpoint is 815M and my pretrained sevila localizer on QVHighlights is 1.4G. Is there any post-processing for the pretrained sevila localizer checkpoints?

**Comment by xijun-cs** [comment]
Hi,

Did you solve this problem? I am in the same situation, and I got the pretrained sevila localizer on QVHighlights with 1.4G too.

Best

**Comment by fake-warrior8** [comment]
> Hi,
> 
> Did you solve this problem? I am in the same situation, and I got the pretrained sevila localizer on QVHighlights with 1.4G too.
> 
> Best

The pretrained ckpt includes BLIP-2 Q-former localizer parameters and some t5 parameters, while the downstream finetuning stage requires only the BLIP-2 Q-form localizer and a original BLIP-2 Q-former answerer parameters. You should combine the pretrained ckpt and the original BLIP-2 parameters to get a new ckpt for downstream finetuning.

**Comment by xijun-cs** [comment]
Thanks for the replying.  Yes, I figured out by printing those keys. And I directly replaced the downloaded ckp's weights related to loc.

**Comment by Domogro** [comment]
Would you help me clarifying this, so sevila has the checkpoints from the localizer and the zero shot answerer, but what about :
"https://storage.googleapis.com/sfr-vision-language-research/LAVIS/models/BLIP2/blip2_pretrained_flant5xl.pth"

This would be the again the original BLIP-2 Q-former?
